---
layout: paper
title: Identification of human silencers by correlating cross-tissue epigenetic profiles and gene expression
authors: Di Huang, Hanna M. Petrykowska, <b>Brendan F. Miller</b>, Laura Elnitski, Ivan Ovcharenko
image:
abstract: <br>Compared to enhancers, silencers are notably difficult to identify and validate experimentally. In search for human silencers, we utilized H3K27me3-DNase I hypersensitive site (DHS) peaks with tissue specificity negatively correlated with the expression of nearby genes across 25 diverse cell lines. These regions are predicted to be silencers since they are physically linked, using Hi-C loops, or associated, using expression quantitative trait loci (eQTL) results, with a decrease in gene expression much more frequently than general H3K27me3-DHSs. Also, these regions are enriched for the binding sites of transcriptional repressors (such as CTCF, MECOM, SMAD4, and SNAI3) and depleted of the binding sites of transcriptional activators. Using sequence signatures of these regions, we constructed a computational model and predicted approximately 10,000 additional silencers per cell line and demonstrated that the majority of genes linked to these silencers are expressed at a decreased level. Furthermore, single nucleotide polymorphisms (SNPs) in predicted silencers are significantly associated with disease phenotypes. Finally, our results show that silencers commonly interact with enhancers to affect the transcriptional dynamics of tissue-specific genes and to facilitate fine-tuning of transcription in the human genome.
journal: Genome Research. March 18, 2019. doi:10.1101/gr.247007.118
journalurl: https://doi.org/10.1101/gr.247007.118
pubmedurl:
pdfurl: pdfs/2019-03-18-silencers_genomeRes.pdf
code:
---
