---
layout: paper
title: Improving the clinical performance of blood-based DNA methylation biomarkers utilizing locus-specific epigenetic heterogeneity
authors: <b>Brendan F. Miller</b>, Thomas R. Pisanic II, Gennady Margolin, Hanna M. Petrykowska, Pornpat Athamanolap, Alexander Goncearenco, Akosua Osei-Tutu, Christina M. Annunziata, Tza-Huei Wang, Laura Elnitski
image:
abstract: <br><i> Background </i> - Variation in intracellular methylation patterns can complicate the use of methylation biomarkers for clinical diagnostic applications such as blood-based cancer testing. Here, we describe development and validation of a methylation density binary classification method called EpiClass (available for download at https://github.com/bmill3r/EpiClass), that can be used to predict and optimize the performance of methylation biomarkers, particularly in challenging, heterogeneous samples such as liquid biopsies. This approach is based upon leveraging statistical differences in single-molecule sample methylation density distributions to identify ideal thresholds for samples classification. <br><i> Results </i> - We developed and tested the classifier using reduced representation bisulfite sequencing (RRBS) data derived from ovarian carcinoma tissue DNA and controls. We used these data to perform in silico simulations using methylation density profiles from individual DNA molecules from ZNF154, a genomic locus known to be recurrently methylated in numerous cancer types. From these profiles, we predicted the performance of the classifier in liquid biopsies for the detection of epithelial ovarian carcinomas (EOC). In silico analysis indicated that EpiClass could be leveraged to better identify cancer-positive liquid biopsy samples by implementing precise thresholds with respect to methylation density profiles derived from circulating cell-free DNA (cfDNA) analysis. These predictions were confirmed experimentally by generating cfDNA methylation density profiles from a cohort of low volume (1-mL) plasma samples obtained from 26 EOC-positive and 41 cancer-free women. EpiClass performance was then validated in an independent cohort of plasma specimens from 24 EOC-positive and 12 cancer-free women, attaining a sensitivity/specificity of 91.7%/100.0%. Direct comparison of CA-125 measurements with EpiClass demonstrated that EpiClass outperformed CA-125 by correctly classifying 69.6% of samples as compared to 47.8% by standard CA-125 assessment. <br><i> Conclusions </i> - Our results indicate that assessment of intramolecular methylation densities calculated from cfDNA facilitate the use of methylation biomarkers for diagnostic applications. Furthermore, we demonstrated that EpiClass analysis of ZNF154 methylation was able to outperform CA-125 in the detection of etiologically-diverse ovarian carcinomas, indicating the broad utility of ZNF154 for use as a biomarker of ovarian cancer.
journal: bioRxiv. December 7, 2019. doi:10.1038/s41592-019-0619-0
journalurl: https://doi.org/10.1101/579839
pubmedurl:
pdfurl: pdfs/2020-12-07-EpiClass_bioRxiv.pdf
code: https://github.com/bmill3r/EpiClass
---
