---
layout: paper
title: Fast, sensitive and accurate integration of single-cell data with Harmony
authors: Ilya Korsunsky, Nghia Millard, <b>Jean Fan</b>, Kamil Slowikowski, Fan Zhang, Kevin Wei, Yuriy Baglaenko, Michael Brenner, Po-ru Loh, and Soumya Raychaudhuri 
image: /assets/papers/nmeth101038.gif
abstract: The emerging diversity of single-cell RNA-seq datasets allows for the full transcriptional characterization of cell types across a wide variety of biological and clinical conditions. However, it is challenging to analyze them together, particularly when datasets are assayed with different technologies, because biological and technical differences are interspersed. We present Harmony (https://github.com/immunogenomics/harmony), an algorithm that projects cells into a shared embedding in which cells group by cell type rather than dataset-specific conditions. Harmony simultaneously accounts for multiple experimental and biological factors. In six analyses, we demonstrate the superior performance of Harmony to previously published algorithms while requiring fewer computational resources. Harmony enables the integration of ~106 cells on a personal computer. We apply Harmony to peripheral blood mononuclear cells from datasets with large experimental differences, five studies of pancreatic islet cells, mouse embryogenesis datasets and the integration of scRNA-seq with spatial transcriptomics data.
journal: Nature Methods. November 18, 2019. doi:10.1038/s41592-019-0619-0
journalurl: https://www.nature.com/articles/s41592-019-0619-0
pubmedurl: https://www.ncbi.nlm.nih.gov/pubmed/
pdfurl: /assets/papers/nmeth101038.pdf
code: https://github.com/immunogenomics/harmony
---
